integration/utils/scripts.go

```diff
@@ -0,0 +1,236 @@
+	return []byte(withAmount), nil
+}
+
+// AddKeyToAccountScript returns a transaction script to add keys to an account
+func AddKeyToAccountScript() ([]byte, error) {
+	return []byte(`
+    transaction(keys: [[UInt8]]) {
+      prepare(signer: AuthAccount) {
+      for key in keys {
+        signer.addPublicKey(key)
+      }
+      }
+    }
+    `), nil
+}
+
+const createAccountsScriptTemplate = `
+import FungibleToken from 0x%s
+import FlowToken from 0x%s
+
+transaction(publicKey: [UInt8], count: Int, initialTokenAmount: UFix64) {
+  prepare(signer: AuthAccount) {
+	let vault = signer.borrow<&FlowToken.Vault>(from: /storage/flowTokenVault)
+      ?? panic("Could not borrow reference to the owner's Vault")
+
+    var i = 0
+    while i < count {
+      let account = AuthAccount(payer: signer)
+      account.addPublicKey(publicKey)
+
+	  let receiver = account.getCapability(/public/flowTokenReceiver)!.borrow<&{FungibleToken.Receiver}>()
+		?? panic("Could not borrow receiver reference to the recipient's Vault")
+
+      receiver.deposit(from: <-vault.withdraw(amount: initialTokenAmount))
+
+      i = i + 1
+    }
+  }
+}
+`
+
+// CreateAccountsScript returns a transaction script for creating an account
+func CreateAccountsScript(fungibleToken, flowToken flowsdk.Address) []byte {
+	return []byte(fmt.Sprintf(createAccountsScriptTemplate, fungibleToken, flowToken))
+}
+
+const myFavContract = `
+access(all) contract MyFavContract {
+
+    init() {
+        self.itemCounter = UInt32(0)
+        self.items = []
+    }
+
+    // items 
+    access(all) event NewItemAddedEvent(id: UInt32, metadata: {String: String})
+
+    access(self) var itemCounter: UInt32
+
+    access(all) struct Item {
+
+            pub let itemID: UInt32
+
+            pub let metadata: {String: String}
+            
+            init(_ metadata: {String: String}) {
+                self.itemID = MyFavContract.itemCounter
+                self.metadata = metadata
+
+                // inc the counter
+                MyFavContract.itemCounter = MyFavContract.itemCounter + UInt32(1)
+
+                // emit event
+                emit NewItemAddedEvent(id: self.itemID, metadata: self.metadata)
+            }
+    }
+
+    access(self) var items: [Item]
+
+    access(all) fun AddItem(_ metadata: {String: String}){
+        let item = Item(metadata)
+        self.items.append(item)
+    }
+
+    access(all) fun AddManyRandomItems(_ n: Int){
+        var i = 0
+        while i < n {
+            MyFavContract.AddItem({"data": "ABCDEFGHIJKLMNOP"})
+            i = i + 1
+        }
+    }
+
+    // heavy operations 
+    // computation heavy function
+    access(all) fun ComputationHeavy() {
+    	var s: Int256 = 1024102410241024
+        var i = 0
+        var a = Int256(7)
+        var b = Int256(5)
+        var c = Int256(2)
+        while i < 15000 {
+            s = s * a
+            s = s / b
+            s = s / c
+            i = i + 1
+        }
```

Comment:
Does this computation really takes a long time?

Reply:
not really long time, about 2 seconds, thats good enough when we combine it with 100 TPS